samples/python/com.ibm.streamsx.topology.pysamples/info.xml

```diff
@@ -3,7 +3,16 @@
  xmlns:info="http://www.ibm.com/xmlns/prod/streams/spl/toolkitInfo">
 <info:identity>
   <info:name>com.ibm.streamsx.topology.pysamples</info:name>
-  <info:description>Sample toolkit for Python support</info:description>
+  <info:description>
+    Sample toolkit for Python support.
+    This sample demonstrates the usage of operators written in python in a Streams spl application.
```

Comment:
nit: "Python" is the name of the language.

Reply:
What does this mean in this concern?
Are there substantial objections?